ember-flight-icons/index.js

```diff
@@ -1,5 +1,13 @@
 'use strict';
 
+const flightIconSprite = require('./public/icons/flight-icon-sprite');
+
 module.exports = {
   name: require('./package').name,
+
+  contentFor(type) {
+    if (type === 'body-footer') {
+      return flightIconSprite;
+    }
+  },
```

Comment:
nice - thanks, i couldn't remember what we had done here

i'm taking a look now